drivers/UARTSerial.h

```diff
@@ -56,6 +56,15 @@ class UARTSerial : private SerialBase, public FileHandle, private NonCopyable<UA
      */
     virtual short poll(short events) const;
 
+    /** Resolve ambiguity versus our private SerialBase */
```

Comment:
Oh, I think these shouldn't be doxygen comments (no double asteriks). The inherited function should pop up in the generated doxygen.

Reply:
Makes sense. A bit of Google suggests doxygen fails to understand `using` declarations, such that this comment doesn't achieve anything, and if we were `using` a private function, it wouldn't show up. But in this case it's a public function that was visible anyway.